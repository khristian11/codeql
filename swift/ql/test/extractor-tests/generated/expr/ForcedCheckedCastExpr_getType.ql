import codeql.swift.elements
import TestUtils

from ForcedCheckedCastExpr x
where toBeTested(x)
select x, x.getType()
