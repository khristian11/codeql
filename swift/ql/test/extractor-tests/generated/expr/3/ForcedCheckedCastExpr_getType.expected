| expressions.swift:36:5:36:12 | (Double) ... | Double |
